tensorflow/compiler/xla/service/gpu/tests/gemm_rewrite_test.cc

```diff
@@ -4863,6 +4863,76 @@ TEST_P(ParameterizedFp8GemmRewriteTest, ScaledABScaledDMatrixBiasF8) {
       )");
 }
 
+TEST_F(CublasLtF8GemmRewriteTest, ScaledABScaledDVectorBiasF8) {
```

Comment:
Add a test similar to VectorBiasThenMatrixBias for FP8. In your case, only the matrix bias would be fused.